selectors/index.js

```diff
@@ -4,6 +4,8 @@ module.exports={
     password_field: '#password',
     otp: '#otp',
     otp_error: '#js-flash-container > div > div',
+    sec_key_tfa_url: 'https://github.com/sessions/two-factor/security-key',
+    otp_tfa_url: 'https://github.com/sessions/two-factor',
```

Comment:
I deleted this line

Reply:
Okay...

Reply:
Like I had mentioned in the below comments (I don't know why my comments are now asynchronous 😂, but I hope you get to understand 🤗), we can't scrap those two URLs because one is for TFA OTP and the other is for TFA security key. So if the challenge is security key, I made the browser to navigate to plain OTP TFA which will then be like the user clicked it there on the security key page, and GitHub will then send the OTP.